### PR TITLE
Accept both single value/multi values in params

### LINKED ISSUE #2173: Unexpected Error message when using .update with just a string
**Description**

Using the new Python deephaven API, getting an unexpected error when using the .update method incorrectly.

**Steps to reproduce**
1. Run:
```
from deephaven import time_table
tt = time_table("00:00:01").update("y=Math.sin(i)")
```

**Expected results**
1. Table is created correctly, or a helpful error message is output

**Actual results**
1. Error message is returned, saying:
```
Type: <class 'deephaven.dherror.DHError'>
Value: table update operation failed. : RuntimeError: java.lang.IllegalArgumentException: Unable to parse formula '=', expected form '<newColumn>=<expression>'
Line: 246
```
This is somewhat confusing, because my formula _does_ seem to be that format. But I think it's because it's expecting an array, and it's treating the string as an array of chars.

### PR DESCRIPTION
**Pull request by jmao-denver**
To make transition to the new Python API easier.

Fixes #2173

### CODE REVIEW

**Review by mofojed — APPROVED** [review]
Looks like it fixes the case in the ticket, was able to run it successfully now.

**Review by rcaudy — APPROVED** [review]

File: pyintegration/tests/test_table.py [hunk 1]
```diff
@@ -139,10 +170,8 @@ def test_where_in(self):
             self.assertLessEqual(unique_table.size, result_table.size)
 
         with self.subTest("where-not-in filter"):
-            result_table2 = self.test_table.where_not_in(unique_table, cols=["c"])
-            self.assertEqual(
-                result_table.size, self.test_table.size - result_table2.size
-            )
```
Comment (chipkent) [inline pyintegration/tests/test_table.py:145]: Did you mean to delete this one?

File: pyintegration/tests/test_table.py [hunk 2]
```diff
@@ -394,7 +463,7 @@ def test_agg_all_by(self):
                 self.assertGreaterEqual(result_table.size, 1)
 
         with self.assertRaises(DHError) as cm:
-            test_table.agg_all_by(count_("aggCount"), ["grp_id"])
+            test_table.agg_all_by(count_("aggCount"), "grp_id")
```
Comment (chipkent) [inline pyintegration/tests/test_table.py:466]: Should both of these be tested?

**Review by chipkent — CHANGES_REQUESTED** [review]

File: pyintegration/tests/test_table.py (hunk 2 above)
Comment (jmao-denver) [reply pyintegration/tests/test_table.py:466]: yes, the single value case covers the original test and this change

File: pyintegration/tests/test_table.py (hunk 1 above)
Comment (jmao-denver) [reply pyintegration/tests/test_table.py:145]: yes, the single value case covers both the original test and this change

**Review by chipkent — APPROVED** [review]

**Pull request merged**